scripts/busybox-ping-test/alcor_services.ini

```diff
@@ -1,20 +1,20 @@
 [services]
 ignite                 = {"name":"ignite", "port":10800, "path":"/lib/ignite.Dockerfile"}
 vpc_manager            = {"name":"vpm",    "port":9001}
-subnet_manager         = {"name":"snm",    "port":9002}
-route_manager          = {"name":"rm",     "port":9003}
-private_ip_manager     = {"name":"pim",    "port":9004}
-mac_manager            = {"name":"mm",     "port":9005}
-port_manager           = {"name":"pm",     "port":9006}
+subnet_manager         = {"name":"snm",    "port":9009}
+route_manager          = {"name":"rm",     "port":9009}
+private_ip_manager     = {"name":"pim",    "port":9009}
+mac_manager            = {"name":"mm",     "port":9009}
+port_manager           = {"name":"pm",     "port":9009}
 node_manager           = {"name":"nm",     "port":9007}
-security_group_manager = {"name":"sgm",    "port":9008}
+security_group_manager = {"name":"sgm",    "port":9009}
 api_gateway            = {"name":"ag",     "port":9009}
-data_plane_manager     = {"name":"dpm",    "port":9010}
-elastic_ip_manager     = {"name":"eim",    "port":9011}
-quota_manager          = {"name":"qm",     "port":9012}
-network_acl_manager    = {"name":"nam",    "port":9013}
-network_config_manager = {"name":"ncm",    "port":9014}
-gateway_manager        = {"name":"gm",     "port":9015}
+data_plane_manager     = {"name":"dpm",    "port":9009}
+elastic_ip_manager     = {"name":"eim",    "port":9009}
+quota_manager          = {"name":"qm",     "port":9009}
+network_acl_manager    = {"name":"nam",    "port":9009}
+network_config_manager = {"name":"ncm",    "port":9009}
+gateway_manager        = {"name":"gm",     "port":9009}
```

Comment:
@pkommoju This is a pretty smart short cut :-) 

In case we need to recover the internal microservice ports, I would recommend to give a comment to each port change like "// subnet internal port : 9002"